docker-compose.yaml

```diff
@@ -48,6 +48,7 @@ services:
       - 3001:3001
     volumes:
       - issuer-server-data:/usr/local/var/keri
+      - ./brans-data:/brans-data
```

Comment:
We should use an actual volume. There is already `issuer-server-data` but not sure why because the server uses Signify and not KERIA/keripy.

Change volumes to: `- issuer-server-data:/path/to/data` and replace with whatever path makes sense in docker